sdk/extensions/ci.yml

```diff
@@ -37,7 +37,13 @@ extends:
       safeName: AzureExtensionsAspNetCoreConfigurationSecrets
     - name: Microsoft.Azure.WebJobs.Extensions.Clients
       safeName: MicrosoftAzureWebJobsExtensionsClients
+    - name: Microsoft.CoreWCF.Azure.StorageQueues
```

Comment:
Let's break WCF into it's own pipelines.   I don't think we want to run it as part of the Azure extensions set.  An example can be found [here](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventhub), where the `ci.yml` is attached to one family of packages and the `ci.data.yml` is attached to another.

We'd want opinions from @weshaggard and @hallipr on naming for the pipelines.  My suggestion would be to use `ci.wcf.yml` and  `tests.wcf.yml`.

Reply:
I would be fine with ci.wcf.yml etc our pipeline generation should support this and create a pipeline named `net - extensions - wcf[ - ci | - tests]`. In order to split like this though you will need a service.project file in the directory, similar to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/service.projects, that will partition the projects.

Reply:
This should be done now. I believe there's an azp command you need to issue to get the pipeline generated.